frontend/www/js/omegaup/components/problem/Mine.vue

```diff
@@ -192,8 +216,11 @@ export default class ProblemMine extends Vue {
   @Prop() privateProblemsAlert!: boolean;
   @Prop() isSysadmin!: boolean;
   @Prop() visibilityStatuses!: Array<string>;
+  @Prop() initialKeyword!: string | null;
```

Comment:
```suggestion
  @Prop() query!: string | null;
```

la convención nueva es ponerle al prop el nombre "normal" y al miembro el nombre con el prefijo `current`. así los otros componentes que lo llaman no tienen que saber si el prop se va a modificar adentro para saber si le tienen que poner el prefijo `initial`.